core/skeleton.py

```diff
@@ -15,7 +15,7 @@
 from viur.core.bones.base import ReadFromClientError, ReadFromClientErrorSeverity, getSystemInitialized
 from viur.core.tasks import CallableTask, CallableTaskBase, QueryIter, CallDeferred
 
-__undefindedC__ = object()
+__undefined__ = object()
```

Comment:
If we're renaming this, can we please call it the common name? (https://github.com/search?l=Python&q=%22sentinel+%3D+object%28%29%22&type=Code)

```suggestion
sentinel = object()
```

Reply:
Ok, we can omit the `__*__` notation. But sentinel isn't a better name. It sais nothing. Why not `UndefinedKindName`? This is equal to `None`.

Reply:
Okay, just `__undefined`? But only with the dunder prefix. _kindname_ should be clear from the context (assignment, comparison).

```suggestion
__undefined = object()
```

Reply:
Ok, please re-review.